build_test/lib/src/package_reader.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:build/build.dart';
+import 'package:glob/glob.dart';
+import 'package:package_resolver/package_resolver.dart';
+import 'package:path/path.dart' as path;
```

Comment:
I would import this `as p`, `path` is a very common local variable or parameter name and it tends to get highly confusing :)